spring-cloud-context/src/test/java/org/springframework/cloud/bootstrap/encrypt/EnvironmentDecryptApplicationInitializerTests.java

```diff
@@ -75,14 +81,22 @@ public void propertySourcesOrderedCorrectly() {
 		assertEquals("spam", context.getEnvironment().getProperty("foo"));
 	}
 
-	@Test(expected = IllegalStateException.class)
+	@Test
 	public void errorOnDecrypt() {
 		this.listener = new EnvironmentDecryptApplicationInitializer(
 				Encryptors.text("deadbeef", "AFFE37"));
 		ConfigurableApplicationContext context = new AnnotationConfigApplicationContext();
 		TestPropertyValues.of("foo: {cipher}bar").applyTo(context);
-		this.listener.initialize(context);
-		assertEquals("bar", context.getEnvironment().getProperty("foo"));
+		// catch IllegalStateException and verify
+		try {
+			this.listener.initialize(context);
+		} catch (Exception e) {
+			assertTrue(e instanceof IllegalStateException);
+		}
+		// Assert logs contain warning even when exception thrown
+		String sysOutput = this.outputCapture.toString();
+		assertThat("Decryption error log message missing", sysOutput, containsString(
+				"EnvironmentDecryptApplicationInitializer - Cannot decrypt: key=foo"));
```

Comment:
Looks like these tests failed because of the ` - ` vs ` : `. Maybe just check for "Cannot decrypt: key=foo"?

Reply:
Removed the class-name.